README.md

```diff
@@ -1,19 +1,46 @@
 # GOV.UK application deployment scripts
 
-This repository contains Capistrano deployment scripts for applications
-running on GOV.UK.
+Capistrano deployment scripts for applications running on GOV.UK.
 
 ## Adding a new app
 
-See [the documentation in the manual](https://docs.publishing.service.gov.uk/manual/setting-up-new-rails-app.html).
+Create a new directory for your app based on one of the other apps e.g.
 
-## How deployments work
+```
+# Capfile
+
+load 'deploy'
+
+$:.unshift(File.expand_path('../../lib', __FILE__))
+load_paths << File.expand_path('../../recipes', __FILE__)
+
+load 'config/deploy'
+```
+
+```
+# config/deploy.rb
 
-The `jenkins.sh` script in this repo is run from the
-[Jenkins job](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_jenkins/templates/jobs/deploy_app.yaml.erb)
-to deploy our applications.
+set :application, "myapp"
+set :capfile_dir, File.expand_path('../', File.dirname(__FILE__))
+set :server_class, "backend"
```

Comment:
Might want to switch this to be a junk one like myapp so it's clearer to change this.

Reply:
Could you expand on that a bit - I'm not sure what you mean :-|

Reply:
Oops sorry - `set :server_class, "backend"` since there are a variety of server_classes I thought it could be clearer specifying backend as a junk one e.g myserver so it's clearer that a user may need to change this setting up a deployment.

I'm not even sure it's that great a suggestion though tbh :-/

Reply:
I think so - people can look at the other configs to find their class.